element_array_ephys/ephys_acute.py

```diff
@@ -91,21 +68,21 @@ def get_ephys_root_data_dir() -> list:
 
 
 def get_session_directory(session_key: dict) -> str:
-    """
-    get_session_directory(session_key: dict) -> str
-        Retrieve the session directory containing the
-         recorded Neuropixels data for a given Session
-        :param session_key: a dictionary of one Session `key`
-        :return: a string for relative or full path to the session directory
+    """Retrieve the session directory with Neuropixels for the given session.
+
+    Args:
+        session_key (dict): A dictionary mapping subject to an entry in the subject table, and session_datetime corresponding to a session in the database. 
+    Returns: 
```

Comment:
Does it render ok with the extra space between last arg and `returns:`? I added that extra linebreak to follow convention

Reply:
Good catch. It wasn't rendering correctly. I've fixed it everywhere in this element.